src/gateway/router/routes/searchRoute.ts

```diff
@@ -15,22 +15,26 @@ export async function searchRoute(ctx: Router.RouterContext) {
     const benchmark = Benchmark.measure();
     const result: any = await gatewayDb.raw(
       `
-          SELECT 1 as sort_order, contract_id, 'pst' as type, '{}'::jsonb as interaction, '' as confirmation_status, pst_ticker, pst_name
+          SELECT 1 as sort_order, contract_id as id, 'pst' as type, '{}'::jsonb as interaction, '' as confirmation_status, pst_ticker, pst_name
           FROM contracts
           WHERE pst_ticker ILIKE ? OR pst_name ILIKE ?
           UNION ALL
-          SELECT 2 as sort_order, contract_id, 'contract' as type, '{}'::jsonb as interaction, '' as confirmation_status, '' as pst_ticker, '' as pst_name
+          SELECT 2 as sort_order, contract_id as id, 'contract' as type, '{}'::jsonb as interaction, '' as confirmation_status, '' as pst_ticker, '' as pst_name
           FROM contracts
           WHERE contract_id ILIKE ?
           GROUP BY contract_id, type
           UNION ALL
-          SELECT 3 as sort_order, interaction_id, 'interaction' as type, interaction, confirmation_status, '' as pst_ticker, '' as pst_name
+          SELECT 3 as sort_order, interaction_id as id, 'interaction' as type, interaction, confirmation_status, '' as pst_ticker, '' as pst_name
           FROM interactions
           WHERE interaction_id ILIKE ?
+          UNION ALL
```

Comment:
how slow is this query? in case of interaction id we have a gin index added (https://towardsdatascience.com/how-gin-indices-can-make-your-postgres-queries-15x-faster-af7a195a3fc5)

Reply:
index added